src/style.css

```
@@ -1 +1,76 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+:root {
+  --colorBlye: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  max-width: 1200px;
+  min-width: 1024px;
+  padding: 0 50px;
+  justify-content: flex-end;
```

```suggestion
  justify-content: space-between;
```
it makes logo and navigation automatically on right places